frontend/src/domain/use_cases/map/updateMapInteractionListeners.ts

```diff
@@ -42,12 +42,12 @@ export const updateMapInteractionListeners = (listener: MapInteractionListenerEn
   }
 }
 
-const openDrawLayerModal = dispatch => {
+const openDrawLayerModal = (dispatch, displayLayersSidebar = true) => {
```

Comment:
suggestion: et ici ça donnerait 
```suggestion
const openDrawLayerModal = (dispatch,  options = { displayLayersSidebar: true }) => {
```

Reply:
voir ma réponse au-dessus